partcad-cli/src/partcad_cli/click/commands/render.py

```
@@ -0,0 +1,137 @@
+    type=click.Choice(
+        [
+            "readme",
+            "svg",
+            "png",
+            "step",
+            "stl",
+            "3mf",
+            "threejs",
+            "obj",
+            "gltf",
+        ]
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)